src/components/layout/index.scss

```diff
@@ -124,6 +124,12 @@ section.layout-container {
           margin-top: 3rem;
         }
 
+        &.page-implementers {
+          h1, h2, h3, h4, h5, h6 { 
+            display: inline-block;
```

Comment:
Headings definitely shouldn't be inline things. Headings are to give a high level structure of the page, not for individual pieces of data.